uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/CopyFileRule.java

```diff
@@ -66,6 +67,7 @@ public class CopyFileRule implements TestRule {
             BrokerCompanyPortal.OLD_COMPANY_PORTAL_APK,
             TeamsApp.TEAMS_APK,
             OutlookApp.OUTLOOK_APK,
+            OneDriveApp.ONEDRIVE_APK,
```

Comment:
should we also add OneDrive app to RemoveFirstPartyAppsTestRule?

Reply:
Yup, good catch